tsconfig.json

```diff
@@ -22,7 +22,9 @@
     "isolatedModules": true,
     "noEmit": true,
     "noFallthroughCasesInSwitch": true,
-    "baseUrl": "src/"
+    "baseUrl": "src/",
+    "types": ["vite-plugin-svgr/client"],
+    "typeRoots": ["./node-modules/@types", "src/common/typedefs"]
```

Comment:
Not sure if this line is necessary. VSCode stopped complaining as soon as I created the file, but according to the TS docs it *should* be necessary